src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -7,20 +7,19 @@
 ;; Schemas
 
 (def Type (schema/enum :string :regex))
-(def Method (schema/enum :get :post :put :delete :head :any))
+(def Methods (schema/enum :get :post :put :delete :head :any))
+(def Method (schema/either Methods [Methods]))
```

Comment:
Might just inline the schema into Rule though, and just refer to multiples as `[Method]` rather than a dedicated schema for it